event/pom.xml

```diff
@@ -45,11 +45,6 @@
             <artifactId>guice</artifactId>
         </dependency>
 
-        <dependency>
```

Comment:
lets not remove joda from airlift/event since there could be downstream modules that still use joda.